src/core/lib/security/security_connector/load_system_roots_fallback.cc

```diff
@@ -21,12 +21,12 @@
 #include <grpc/slice_buffer.h>
 #include "src/core/lib/security/security_connector/load_system_roots.h"
 
-#ifndef GPR_LINUX
+#if !defined(GRP_LINUX) && !defined(GPR_ANDROID)
```

Comment:
Should this be GPR_LINUX, not GRP_LINUX?

Reply:
That's a very polite way of putting it, thanks.